README.md

```diff
@@ -39,19 +39,19 @@ You will also need an Amazon Developer account: https://developer.amazon.com
 ## Hardware Setup
 Set up your hardware by following the **Hardware Setup Guide** for your product.
 
-   XVF3510: https://www.xmos.ai/file/xvf3510-dev-kit-setup-guides  
-   XVF3500: https://www.xmos.ai/file/xvf3500-dev-kit-setup-guides  
-   XVF3100: https://www.xmos.ai/file/xvf3000-3100-dev-kit-setup-guides  
-   XVF3000: https://www.xmos.ai/file/xvf3000-3100-dev-kit-setup-guides  
+   XVF3510: https://www.xmos.ai/file/xvf3510-dev-kit-setup-guides
+   XVF3500: https://www.xmos.ai/file/xvf3500-dev-kit-setup-guides
+   XVF3100: https://www.xmos.ai/file/xvf3000-3100-dev-kit-setup-guides
+   XVF3000: https://www.xmos.ai/file/xvf3000-3100-dev-kit-setup-guides
 
 ## Firmware Upgrade
-Once the hardware is setup, upgrade the firmware on your AVS development kit. The firmware can be found here:  
+Once the hardware is setup, upgrade the firmware on your AVS development kit. The firmware can be found here:
+
```

Comment:
I'm a bit surprised that no entries exist here for XVF3615 and XVF3610.  Is that because they are not yet available on the web site?

Reply:
Tom told me to remove 3600 and 3610 few weeks ago, so I didn't add 3615. I am not sure if circumstances changed now.

Reply:
Okay.  I trust that Tom had a good reason for his request.